src/portable_python/cpython.py

```diff
@@ -211,14 +241,45 @@ def _finalize(self):
                 runez.write(self.install_folder / info_path, contents)
 
         PPG.config.cleanup_folder(self, "cpython-clean-2nd-pass", "cpython-clean")
+        self._lock_site_packages()
 
         py_inspector = PythonInspector(self.install_folder)
         print(py_inspector.represented())
         problem = py_inspector.full_so_report.get_problem(portable=not is_shared)
         runez.abort_if(problem and self.setup.x_debug != "direct-finalize", "Build failed: %s" % problem)
 
-    def _check_venv(self, copies=False):
-        """Verify that the freshly compiled python can create venvs without issue"""
+    def _lock_site_packages(self):
+        """Mark site-packages as read-only, this will force any '-mpip install' to imply --user"""
```

Comment:
Please expand.  For example

The default behavior of pip is to imply `--user` installs if the global site-packages isn't writable  Portable python installs are normally user-owned, thus it would be writable so let's fix that here.  The decision is made in pip's `decide_user_install` checking something simple (on *nix, it just uses `os.access(..., os.W_OK)`.

In the words of PEP 668, we want to consider this a "user Python" but haven't adopted the EXTERNALLY-MANAGED file from that (because, quite simply, nothing else is managing it).